### ISSUE TITLE
issue wrt convert_to_bool in latest master

### ISSUE THREAD

**Issue by fooelisa** [Juniper/ansible-junos-stdlib#472]
Issue Type
------
 - Bug Report

Module Name
------
- juniper_junos_rpc (underlying issue in juniper_junos_common)

Juniper.Junos role and Python libraries version
------
```
ansible 2.9.2
  config file = /home/elisa/viki/ansible/ansible.cfg
  ansible python module location = /usr/local/lib/python3.5/dist-packages/ansible
  executable location = /usr/local/bin/ansible
  python version = 3.5.2 (default, Oct  8 2019, 13:06:37) [GCC 5.4.0 20160609]

junos-eznc              2.2.1
jsnapy                       1.3.2
Juniper.junos latest master (date: 12/12/19)
```

OS / Environment
------
- 18.4R2-S1.4 on MX104

Summary
------
The latest changes to juniper_junos_common (wrt convert_to_bool) seem to have broken the ability to pass string kwargs to RPCs.

Steps to reproduce
------
```yaml
- name: sync scripts between RE's
  juniper_junos_rpc:
    rpc: request-scripts-synchronize
    kwargs:
      type: all
```

Results
------
```
TASK [scripts-sync : sync scripts between RE's] *****************************************************************************************************************************************************************************************
task path: /home/elisa/viki/ansible/roles/scripts-sync/tasks/main.yml:3
Using module file /opt/ansible-junos-stdlib/Juniper.junos/library/juniper_junos_rpc.py
Pipelining is enabled.
<esr1.sfo3> ESTABLISH LOCAL CONNECTION FOR USER: elisa
<esr1.sfo3> EXEC /bin/sh -c '/usr/bin/python3 && sleep 0'
The full traceback is:
Traceback (most recent call last):
  File "<stdin>", line 102, in <module>
  File "<stdin>", line 94, in _ansiballz_main
  File "<stdin>", line 40, in invoke_module
  File "/usr/lib/python3.5/runpy.py", line 196, in run_module
    return _run_module_code(code, init_globals, run_name, mod_spec)
  File "/usr/lib/python3.5/runpy.py", line 96, in _run_module_code
    mod_name, mod_spec, pkg_name, script_name)
  File "/usr/lib/python3.5/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/tmp/ansible_juniper_junos_rpc_payload_ai890x2z/ansible_juniper_junos_rpc_payload.zip/ansible/modules/juniper_junos_rpc.py", line 624, in <module>
  File "/tmp/ansible_juniper_junos_rpc_payload_ai890x2z/ansible_juniper_junos_rpc_payload.zip/ansible/modules/juniper_junos_rpc.py", line 461, in main
  File "/tmp/ansible_juniper_junos_rpc_payload_ai890x2z/ansible_juniper_junos_rpc_payload.zip/ansible/module_utils/juniper_junos_common.py", line 1176, in parse_arg_to_list_of_dicts
  File "/tmp/ansible_juniper_junos_rpc_payload_ai890x2z/ansible_juniper_junos_rpc_payload.zip/ansible/module_utils/parsing/convert_bool.py", line 26, in boolean
TypeError: The value 'all' is not a valid boolean.  Valid booleans include: 0, 1, 'off', 'no', 't', 'y', 'yes', '0', '1', 'n', 'false', 'on', 'true', 'f'

fatal: [esr1.sfo3]: FAILED! => {
    "changed": false,
    "module_stderr": "Traceback (most recent call last):\n  File \"<stdin>\", line 102, in <module>\n  File \"<stdin>\", line 94, in _ansiballz_main\n  File \"<stdin>\", line 40, in invoke_module\n  File \"/usr/lib/python3.5/runpy.py\", line 196, in run_module\n    return _run_module_code(code, init_globals, run_name, mod_spec)\n  File \"/usr/lib/python3.5/runpy.py\", line 96, in _run_module_code\n    mod_name, mod_spec, pkg_name, script_name)\n  File \"/usr/lib/python3.5/runpy.py\", line 85, in _run_code\n    exec(code, run_globals)\n  File \"/tmp/ansible_juniper_junos_rpc_payload_ai890x2z/ansible_juniper_junos_rpc_payload.zip/ansible/modules/juniper_junos_rpc.py\", line 624, in <module>\n  File \"/tmp/ansible_juniper_junos_rpc_payload_ai890x2z/ansible_juniper_junos_rpc_payload.zip/ansible/modules/juniper_junos_rpc.py\", line 461, in main\n  File \"/tmp/ansible_juniper_junos_rpc_payload_ai890x2z/ansible_juniper_junos_rpc_payload.zip/ansible/module_utils/juniper_junos_common.py\", line 1176, in parse_arg_to_list_of_dicts\n  File \"/tmp/ansible_juniper_junos_rpc_payload_ai890x2z/ansible_juniper_junos_rpc_payload.zip/ansible/module_utils/parsing/convert_bool.py\", line 26, in boolean\nTypeError: The value 'all' is not a valid boolean.  Valid booleans include: 0, 1, 'off', 'no', 't', 'y', 'yes', '0', '1', 'n', 'false', 'on', 'true', 'f'\n",
    "module_stdout": "",
    "msg": "MODULE FAILURE\nSee stdout/stderr for the exact error",
    "rc": 1
}
```

Workaround
------
Calling `junos_module.parse_arg_to_list_of_dicts` in `juniper_junos_rpc` with `allow_bool_values=False` skips the conversion and fixes it for me in this particular instance. But I am not sure about cases where actual bool values are passed into kwargs, that might in fact need the conversion.

**Comment by vnitinv** [comment]
@fooelisa Elisa Looking into this.

**Comment by rsmekala** [comment]
@fooelisa Merged via #469. Please clone the latest master, it is already fixed.
